customer-wallet/src/containers/Settings/index.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react'
```

Comment:
I like that you moved Settings to a unified page. Earlier designs tried to have separate views for vendors and customers, now they are going to be unified. Show/Hide based on a bit of context - that is the max we'll do to separate the views.

Reply:
One small thing @fazo96 @mcsweenb 
I am not checking out the branch and running it. It is just a quick look at what changes are being made to be honest at the moment.

It will be great if we can include the screenshot of the page you are wrapping up with the PR. Just for next time onwards. Merging this.